hack/windows/Install-OVS.ps1

```diff
@@ -0,0 +1,131 @@
+Param(
+    [parameter(Mandatory = $false)] [string] $DownloadDir,
+    [parameter(Mandatory = $false)] [string] $DownloadURL,
+    [parameter(Mandatory = $false)] [string] $OVSInstallDir = "C:\openvswitch"
+)
+
+$ErrorActionPreference = "Stop"
+$OVSDownloadURL = "https://github.com/ruicao93/ovs/releases/download/v2.12.9-win64/ovs-2.12.90-win64.zip"
```

Comment:
Will you change the url before merging the PR?

Reply:
Yes, this URL will be updated before merge.

I will keep the conversation open to track the issue.